samples/CognitiveSearch/CognitiveSearch/Generated/Models/Analyzer.cs

```diff
@@ -30,7 +30,7 @@ public Analyzer(string name)
         /// <param name="name"> The name of the analyzer. It must only contain letters, digits, spaces, dashes or underscores, can only start and end with alphanumeric characters, and is limited to 128 characters. </param>
         internal Analyzer(string odataType, string name)
         {
-            OdataType = odataType ?? null;
```

Comment:
Should either of these actually be validating? These are both required in Search. Could be separate issue if so.

Reply:
This is a serialization ctor, we are not validating service responses.